projects/example-project-webpack/src/index.js

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+// import our stylesheets
+import './styles.css';
+
+// import the components we'll use in this page
+import '@spectrum-web-components/button/sp-button';
+import '@spectrum-web-components/dropdown/sp-dropdown';
+import '@spectrum-web-components/menu/sp-menu';
+import '@spectrum-web-components/menu/sp-menu-item';
+
+/**
+ * Uncomment this following line to force webpack to include dynamically
+ * imported code powering the `<sp-dropdown>` in the main JS bundle.
+ **/
+// export { Overlay } from '@spectrum-web-components/overlay';
```

Comment:
By default this will structure chunks like so:
![image](https://user-images.githubusercontent.com/1156657/89313789-f4b85f80-d646-11ea-92bd-b9eb59f237e8.png)
![image](https://user-images.githubusercontent.com/1156657/89313818-fe41c780-d646-11ea-915a-ca24b2a4df67.png)

When it is uncommented, you'll receive the single bundle build, like so:
![image](https://user-images.githubusercontent.com/1156657/89313687-d5213700-d646-11ea-9e79-dd456e533b15.png)